### PR TITLE
Add Changelog for v0.19.7

### PR DESCRIPTION
**Pull request by sp1999**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/whatsnew/changelog.md [hunk 1]
```diff
@@ -31,6 +31,36 @@ description: >
...
+### Upgraded
+- Cluster API Provider Nutanix: `v1.3.3` to `v1.3.5`
+- Image Builder: `v0.1.24` to `v0.1.26`
+- EKS Distro:
+  - `v1-25-eks-39` to [`v1-25-eks-40`](https://distro.eks.amazonaws.com/releases/1-25/40/)
+  - `v1-26-eks-35` to [`v1-26-eks-38`](https://distro.eks.amazonaws.com/releases/1-26/38/)
+  - `v1-27-eks-29` to [`v1-27-eks-32`](https://distro.eks.amazonaws.com/releases/1-27/32/)
+  - `v1-28-eks-22` to [`v1-28-eks-25`](https://distro.eks.amazonaws.com/releases/1-28/25/)
+  - `v1-29-eks-11` to [`v1-29-eks-14`](https://distro.eks.amazonaws.com/releases/1-29/14/)
+
```
Comment (rahulbabu95) [inline docs/content/en/docs/whatsnew/changelog.md:55]: Am assuming the previous versions on these just changed by 1.?

Comment (sp1999) [reply docs/content/en/docs/whatsnew/changelog.md:55]: https://distro.eks.amazonaws.com/#eks-d-130-version-dependencies
They had 3 releases on May 28, May 31, June 7

**Pull request merged**